ecu/auth.go

```diff
@@ -27,14 +27,35 @@ import (
 	"github.com/google/seesaw/common/ipc"
 )
 
-// authInit initialises the authentication system.
-func (e *ECU) authInit() error {
+// Authenticator types provide functionality for authenticating users and adding
+// the authenticated information to a context.
+type Authenticator interface {
```

Comment:
If I understand correctly, IPC connection can use trusted context to bypass all the authentications. For any RPC connection to ECU, an authenticator is applied and default is rejecting all. There is no other authenticator implemented yet. is that right?

Reply:
Yes.